jaq-parse/src/token.rs

```diff
@@ -106,6 +106,8 @@ pub fn token() -> impl Parser<char, Token, Error = Simple<char>> {
 
     let var = just('$').ignore_then(text::ident());
 
+    let format_filter = just('@').chain(text::ident()).collect();
```

Comment:
Could you use `just('@').or_not()` to make this more general and then use it instead of text::ident() to define ident below?